internal/environment/get_env_value.go

```diff
@@ -80,6 +81,10 @@ func getLogMessage(env string, messageMap map[string]string) string {
 	return messageMap[defaultValueKey]
 }
 
+func GetEnvValueString(env string) (string, bool) {
```

Comment:
tbh i'd just inline it, this seems like an unnecessary redirection

Reply:
it ended up only being used once, so I'll remove it